scripts/build_osm_network.py

```diff
@@ -726,6 +852,14 @@ def built_network(inputs, outputs, geo_crs, distance_crs):
 
     substations = gpd.read_file(inputs["substations"])
     lines = gpd.read_file(inputs["lines"])
+    links = read_geojson(inputs["links"])
```

Comment:
I'd use the "converters" word instead of "links" also in the code, where they are intended to be converters

Reply:
I think it's definitely make sense to replace links with converters in the outputs of `base_network` and inputs of `build_osm_network`. 

It seems to be a little misleading to have the `links` in `clean_osm_data` as well, while we are going to model them as lines. But it feels a bit too early to use there a `converters` name as the converters will introduced only further (in `build_osm_network`). Probably, `lines_dc` would be more appropriate?

Reply:
I absolutely agree that it's better to have a clean data structure. Removed links both from snakemake inputs and the code.